design/apidsl/attribute.go

```diff
@@ -282,6 +286,7 @@ func Default(def interface{}) {
 	}
 }
 
+// Example can be used in: Attribute
```

Comment:
All the attribute functions below (`Example` to `MaxLen`) can all be used in `Attribute`, `Header`, `Param`, `HashOf` or `ArrayOf`